pageserver/src/tenant/timeline.rs

```diff
@@ -5358,6 +5358,90 @@ impl Timeline {
             shard_count: self.tenant_shard_id.shard_count,
         }
     }
+
+    #[cfg(test)]
+    pub(super) fn force_advance_lsn(self: &Arc<Timeline>, new_lsn: Lsn) {
+        self.last_record_lsn.advance(new_lsn);
+    }
+
+    /// Force create an image layer and place it into the layer may. DO NOT use this function directly. Use `branch_timeline_test_with_states`
+    /// or `create_test_timeline_with_states` to ensure all these layers are placed into the layer map in one run.
```

Comment:
```suggestion
    /// Force create an image layer and place it into the layer may. DO NOT use this function directly. Use `branch_timeline_test_with_states`
    /// or `create_test_timeline_with_states` to ensure all these layers are placed into the layer map in one run.
```
```suggestion
    /// Force create an image layer and place it into the layer map.
    ///
    /// DO NOT use this function directly. Use `branch_timeline_test_with_states`
    /// or `create_test_timeline_with_states` to ensure all these layers are placed into the layer map in one run.
```

- may => map
- added parabreak

Additionally if you made these function calls into links *before* you renamed them, then you'd get a `cargo doc --document-private-items` warnings, so please make them into links.